app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -27,8 +27,12 @@ class SensorReceiver : BroadcastReceiver() {
             LightSensorManager(),
             NextAlarmManager(),
             PhoneStateSensorManager(),
-            StorageSensorManager()
+            StorageSensorManager(),
+            ActivitySensorManager()
```

Comment:
Maybe move this to the top so the list remains alphabetical? :)

Reply:
Sure, didn't notice that

Reply:
After my comment I realized that this list wasn't completely alphabetical to begin with 🙈 sorry about that but thanks for correcting it nonetheless :)